PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+###### Explanation About What Code Achieves:
```

Comment:
What's the reasoning behind using H6? Seems a little too deep to me. [H4s](https://gist.github.com/evnm/0fc670b2f7c55962a0c21254191a1585) have a similar weight and would allow folks to still use sub-headers.

Reply:
H6 are shrunk and gray in github
H4 are normal sized and bold

I generally use H6 so that people can tell the difference between the template and the text that the user has entered